application/src/main/java/org/togetherjava/tjbot/commands/example/AbstractCommand.java

```diff
@@ -9,6 +9,10 @@
  * This helps against a tiny bit of duplicated code.
  */
 public class AbstractCommand implements Command {
+    /**
+     * The commandName is the actual command message that the user will/can call to call the Command
```

Comment:
There was a PR for adding docs to the constructor.
These variables are private so no-one will see these docs, they're also useless and lack of information.